roles/overpass-api/templates/overpass-areas-builder.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Overpass areas builder
+Requires=overpass-dispatcher.service overpass-areas-dispatcher.service
+AssertPathExists={{ overpass_database_dir }}/replicate_id
+
+[Service]
+Type=simple
+User={{ user }}
+ExecStart=/usr/local/bin/rules_loop.sh {{ overpass_database_dir }}
+Nice=19
+Restart=always
+RestartSec=30s
```

Comment:
This job will run for several hours (delta mode is unavailable due to lack of attic data). Maybe have a longer wait time in case job causes too much load.